baselines/fedpara/pyproject.toml

```diff
@@ -0,0 +1,141 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "fedpara" # <----- Ensure it matches the name of your baseline directory containing all the source code
+version = "1.0.0"
+description = "Flower Baselines"
+license = "Apache-2.0"
+authors = ["Yahia Salaheldin Shaaban <> ", "Omar Mokhtar <>", "Roeia Amr <>"]
```

Comment:
I formatted this slightly. If you wish to provide your contact email please do so, else remove the `<>` part.